docker/hq-compose-runserver.yml

```diff
@@ -21,6 +21,22 @@ web:
     - ..:/mnt/commcare-hq-ro:ro
     - hqservice-lib:/mnt/lib
 
+formplayer:
+  extends:
+    file: hq-compose.yml
+    service: formplayer
+  environment:
+    DEPENDENT_SERVICES: "COUCH:5984 POSTGRES:5432 REDIS:6379"
+    WEB_HOST: "web"
+  links:
+    - postgres
+    - couch
+    - redis
+  expose:
+    - 8080
```

Comment:
would it be possible/acceptable to use a different port? I use 8080 locally for nginx, and I'd rather not change it if I don't have to.

Reply:
I think first prize would be to have it configurable using an environment variable, but this port is determined by FormPlayer's configuration set in (or used by) its Dockerfile, so buildtime not runtime. So I think we'll have to set a fixed port number. (And if there is a better way, I'd love to know it.)

8081? 9080? 

fwiw, I vote 8081.

Reply:
haha, I use 8081 locally as well. I pretty much assume 808X is reserved for my personal workspace, so something outside of that range? How about 8010? It [seems to be free](https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers) and the HQ codebase doesn't seem to reference it anywhere that I can see.